ui/admin/app/templates/scopes/scope/session-recordings/session-recording/channels-by-connection/index.hbs

```diff
@@ -26,6 +30,17 @@
             (route-action 'reapplyStoragepolicy' @model.sessionRecording)
           }}
         />
+        {{#if (eq @model.sessionRecording.state 'available')}}
```

Comment:
Can we only delete sessions that are available? If so, should we make this part of the ability check too?

Reply:
that's a good point, I can make that change